wfe2/wfe.go

```diff
@@ -2326,6 +2349,12 @@ func (wfe *WebFrontEndImpl) NewOrder(
 		}
 	}
 
+	err = wfe.validateCertificateProfileName(newOrderRequest.Profile)
+	if err != nil {
+		wfe.sendError(response, logEvent, probs.Malformed("Invalid certificate profile, %q: %s", newOrderRequest.Profile, err), err)
```

Comment:
Leave a TODO so that we can place a letsencrypt.org doc page here for available profiles, when that page gets written.